drivers/flash/flash_stm32_qspi.c

```diff
@@ -292,18 +483,43 @@ static int flash_stm32_qspi_write(const struct device *dev, off_t addr,
 		return -EINVAL;
 	}
 
-	QSPI_CommandTypeDef cmd_write_en = {
-		.Instruction = SPI_NOR_CMD_WREN,
+#ifndef STM32_OSPI_MIMIC_QSPI
+#if DT_INST_PROP(0, enable_quad_output)
```

Comment:
Decided with @JvanDooren that this will become too complex in the other way since a `#define` will be required for each off the variables, for both single and quad, and for both QSPI and OSPI. Therefore ignoring this comment.